app/src/main/java/com/github/palFinderTeam/palfinder/navigation/MainNavActivity.kt

```diff
@@ -138,17 +137,12 @@ class MainNavActivity : AppCompatActivity() {
                         } else {
                             navController.popBackStack()
                             val args = Bundle().apply {
-                                putSerializable("showParam", ShowParam.ONLY_JOINED)
+                                putSerializable("UserId", loggedUser)
```

Comment:
Hard coded string =)

Reply:
yeah the problem is that they are defined in the navigation and I still haven't found a way to access  them programmatically (it's also the case for the meetup creation compat) but I could always add constants.